aixplain/modules/metric.py

```diff
@@ -43,23 +43,40 @@ def __init__(
         self,
         id: Text,
         name: Text,
-        description: Text,
-        supplier: Text = "aiXplain",
-        version: Text = "1.0",
```

Comment:
Please keep the version. It is important to keep the difference across some metrics (e.g. COMET DA 2020, COMET DA 2021, etc.)